File: cs/src/formulas/matrixScalaMultiplation.cs
public static class MatrixScalaMultiplation
{
    public static double[,] Multiply(double[,] matrix, double k)
    {
        int rows = matrix.GetLength(0);
        int cols = matrix.GetLength(1);
            double[,] res = new double[rows, cols];
            for (int i = 0; i < rows; i++)
            {
                for (int j = 0; j < cols; j++)
                {
                    res[i, j] = matrix[i, j] * k;
                }
            }
            return res;
    }
}

File: py/src/formulas/matrixScalaMultiplation.py
def matrix_scala_multiplation(matrix: list[list[float]], k: float) -> list[list[float]]:
    res = []
    for i in range(len(matrix)):
        res.append([])
        for j in range(len(matrix[0])):
            res[i].append(matrix[i][j] * k)
    return res
